outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -247,6 +260,78 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
         );
     }
 
+    private getCompanyEnrichmentAvailableSection = () => {
+        if (this.state.company.isAddedToDatabase()) {
+            return (
+                <div>
+                    {this.getCompanyInsightsSection()}
+                    <div>
+                        <div className='odoo-secondary-button insights-button'
+                             onClick={this.enrichAndUpdate}>
+                            {_t("Enrich Company")}
+                        </div>
+                    </div>
+                </div>
+            );
+        } else {
+            return (
+                <div className='muted-text insights-info' >
+                    {_t("No company attached to this contact")}
+                    <div className='odoo-secondary-button insights-button'
+                         onClick={this.enrichAndCreate}>
+                        {_t("Create a Company")}
+                    </div>
+                </div>
+            );
+        }
+    }
+
+    private getCompanyEnrichmentEmptySection = () => {
+        let insightsSection = null;
+        if (this.context.isConnected()) {
+            if (this.state.company.isAddedToDatabase()) {
+                return (
+                  <div>
+                      {this.getCompanyInsightsSection()}
+                      <div className='muted-text insights-info'>
+                          {_t("No additional insights were found for this company")}
```

Comment:
Check for translations (also for other strings in this file) as it does not seem to be currently available in translation files

Reply:
Same thing, must be added in the forward port.